### DISCUSSION TITLE
What is "forge doc"?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [foundry-rs/foundry#3996] [Help] [answered]
I saw this command mentioned in the [Best Practices](https://book.getfoundry.sh/tutorials/best-practices?highlight=forge%20doc#comments).

What exactly does it do and how are we supposed to use it?

If I run `forge doc`, it says that the command was unrecognized. Also, running `forge --help` doesn't mention the `doc` command.

I am using the latest Forge at the time of creating this discussion, i.e. `forge 0.2.0 (cd8da50 2022-12-30T11:15:05.784035Z)`.

Is this a not yet released feature?

**Comment by mds1** [comment, marked as answer]
It's still a WIP, you can find the PR here: https://github.com/foundry-rs/foundry/pull/2701

It should be in a decent state to test, so feel free to give it a try and leave feedback/feature requests, cc @rkrasiuk

**Comment by rkrasiuk** [comment]
@paulrberg yup, what @mds1 said above. it has not been a prio, so I mostly added to that PR when I had some time to spare. however, i'll focus some time in the following days to take it over the line
